src/main/java/uk/gov/companieshouse/api/strikeoffobjections/validation/AllowedValuesValidationRule.java

```diff
@@ -22,7 +22,7 @@ public AllowedValuesValidationRule(List<T> allowableValues,
     @Override
     public void validate(T input, String logContext) throws ValidationException {
         if (!allowableValues.contains(input)) {
-            apiLogger.debugContext(logContext, String.format("%s is not an allowed value", input));
+            apiLogger.debugContext(logContext, String.format("AllowedValuesValidationRule %s is not an allowed value", input));
```

Comment:
You could use Class.getSimpleName() here instead of repeating the class name. Same in the other rule log statement.

Reply:
updated to use getSimpleName()